its/projects/NuGet.config

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+      <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+  </packageSources>
+  <trustedSigners>
+    <repository name="nuget.org" serviceIndex="https://api.nuget.org/v3/index.json">
+        <certificate fingerprint="0e5f38f57dc1bcc806d8494f4f90fbcedd988b46760709cbeec6f4219aa6157d" hashAlgorithm="SHA256" allowUntrustedRoot="false" />
+        <certificate fingerprint="5a2901d6ada3d18260b9c6dfe2133c95d74b9eef6ae0e5dc334c8454d1477df4" hashAlgorithm="SHA256" allowUntrustedRoot="false" />
+        <!-- StyleCop.Analyzers: sharwell -->
+        <!-- NUnit: charliepoole;OsirisTerje;rprouse -->
+        <!-- coverlet.collector: tonerdo, dotnetfoundation, MarcoRossignoli, daveMueller -->
+        <!-- Microsoft.VisualStudio.UnitTesting: lizoc -->
+        <!-- NUnit.Analyzers: mikkelbu -->
+        <!-- xunit.core dotnetfoundation xunit -->
```

Comment:
This doesn't seem to be maintained in this specific file anymore. So I would remove it.
_Optional:_ Replace it with a single comment explaining that those are owners of packages references by our IT projects
```suggestion
```

Reply:
I wanted to delete the comments, not the file. Because it doesn't match the current list of owners. For example `protobuf-packages`, `castleproject`, `jonorossi` and few others aren't listed.

Reply:
I will add comments for the missing owners.